changelogs/1.31.0.yaml

```diff
@@ -535,6 +535,11 @@ new_features:
     QUIC stream reset error code will be added to transport failure reason.
     This behavior can be reverted by setting the runtime flag ``envoy.reloadable_features.report_stream_reset_error_code``
     to ``false``.
+- area: basic_auth
```

Comment:
why are you changing a historical changelog? the changelogs are immutable you need to add any current changes to `changelogs/current.yaml`

Reply:
oops thought that `v1.31` it not released yet

Reply:
hah, I didn't check the change log file name... 🤣